src/network/networkd-routing-policy-rule.c

```diff
@@ -1016,11 +1025,13 @@ int manager_rtnl_process_rule(sd_netlink *rtnl, sd_netlink_message *message, Man
                 return 0;
         }
 
-        r = sd_netlink_message_read_u8(message, FRA_L3MDEV, &tmp->l3mdev);
+        uint8_t l3mdev;
+        r = sd_netlink_message_read_u8(message, FRA_L3MDEV, &l3mdev);
         if (r < 0 && r != -ENODATA) {
                 log_warning_errno(r, "rtnl: could not get FRA_L3MDEV attribute, ignoring: %m");
                 return 0;
         }
+        tmp->l3mdev = l3mdev & 1;
```

Comment:
really `& 1`? Not rather `!= 0` or so?

Reply:
Since the only valid value is 1? I don't have strong opinions.

Reply:
i mean, if i see `& 1` i get the sense this was not a boolean but a flags field. it's just a bit confusing, that's all